spreadsheets/functions-and-formulas/comparison-operators-and-functions/what-are-comparison-operators.md

```diff
@@ -80,16 +81,16 @@ What are the possible outputs when using comparison operators or their equivalen
 
 Match the function with its equivalent comparison operator.
 
-`<`  = ???
-`=`  = ??? 
-`>`  = ???
-`>=` = ???
-`<>` = ???
-`<=` = ???
-
-- LT()
-- EQ()
-- GT()
-- GTE()
-- NE()
-- LTE()
+`<`  = ???()
+`=`  = ???()
+`>`  = ???()
+`>=` = ???()
+`<>` = ???()
+`<=` = ???()
+
+- LT
+- EQ
+- GT
+- GTE
+- NE
+- LTE
```

Comment:
Curious why this change? Does it look better in the app this way?

Reply:
For type in the gap. Users might forget to type the `()` and it would give them a "wrong answer"

Reply:
But both LT and LT() are technically correct. So this way there can be no mistakes.  regarding that. If they write lowercase it is incorrect anyway as all functions in spreadsheets are full caps